binding/nodejs/index.test.js

```diff
@@ -158,6 +203,122 @@ describe("manual paths", () => {
   });
 });
 
+describe("keyword detection in DE", () => {
+  test("single keyword single detection", () => {
+    let porcupineEngine = new Porcupine(
+      ACCESS_KEY,  
+      keywordPathsSingleDe,
+      SENSITIVITIES_1,
+      MODEL_PATH_DE
+    );
+
+    let counts = porcupineDetectionCounts(
```

Comment:
for the `porcupineDetectionCounts`, we should do the same as in python with paths:

```javascript
const path = require("path")

let some_path = path.join(__dirname, "path_relative_to_this_directory")
```